docs/console-virtual-terminal-sequences.md

```diff
@@ -262,12 +262,12 @@ The following command allows the application to set the screen colors palette va
 
 The RGB values should be hexadecimal values between `0` and `ff`, and separated by the forward-slash character (e.g. `rgb:1/24/86`).
 
-Note that this sequence is an OSC “Operating system command” sequence, and not a CSI like many of the other sequences listed, and as such start with “\\x1b\]”, not “\\x1b\[”.
+Note that this sequence is an OSC “Operating system command” sequence, and not a CSI like many of the other sequences listed, and as such start with “\\x1b\]”, not “\\x1b\[”. As OSC sequences, they are ended with a _String Terminator_ represented as `<ST>` for which either `BEL` (`0x7`) or `ESC \` (`0x1B 0x5C`) are valid terminator sequences.
```

Comment:
nit: let's disrecommend BEL. It is nonstandard. It _works_, but we should explicitly prefer ST in all documentation.

Reply:
that's my B, I was the one that said we should put both there. That's how I always terminate my OSC's 😋

Reply:
Can I say "The preferred terminator is ESC \, but BEL still works?" or something? Or maybe put it as a footnote or something?

Reply:
sure!

technically, ONLY ESC-Slash is `ST`.

Reply:
OK FINE. Try this wording. Yay or nay?